tools/antithesis-go-instrumentor/antithesis-go-instrumentor.go

```diff
@@ -0,0 +1,227 @@
+package main
```

Comment:
What would you think about making a couple subdirectories/go subpackages here? One for assertions, one for instrumentation. And then keep the control flow stuff here in the main directory.

Reply:
Done - and added reformat.sh to run `gofmt` on all source packages